project/Tests/CoreTests/JekyllTests.swift

```diff
@@ -4,18 +4,17 @@ import XCTest
 @testable import NefCore
 
 class JekyllTests: XCTestCase {
```

Comment:
We may want to consider to rewrite these tests as snapshot tests, or combine them, as they are quite prone to change if we add spaces, new lines or such. But they look much nicer now anyways!

Reply:
Sure, after the refactor we could think about how to create a robust-base test code :)